backend/src/packages/auth/helpers/token/token-helper.helper.ts

```diff
@@ -0,0 +1,15 @@
+import jwt from 'jsonwebtoken';
+
+import { type ITokenPayload } from '~/libs/interfaces/interfaces.js';
+import { config } from '~/libs/packages/config/config.js';
+
+const createToken = (payload: { id: number }): string => {
+  return jwt.sign(payload, config.ENV.JWT.SECRET, {
+    expiresIn: config.ENV.JWT.EXPIRES_IN,
+  });
+};
+
+const verifyToken = (token: string): ITokenPayload =>
+  jwt.verify(token, config.ENV.JWT.SECRET) as ITokenPayload;
+
+export { createToken, verifyToken };
```

Comment:
Should be a Token class in `src/libs/packages/token` with implementation of `create`, `verify`, `decode` token methods using `jose`